pkgs/development/python-modules/rns/default.nix

```diff
@@ -34,6 +36,12 @@ buildPythonPackage rec {
   # Module has no tests
   doCheck = false;
 
+  patches = [
+    (replaceVars ./fix-generated-shebang.patch {
+      python = lib.getExe (python.withPackages (ps: [ ps.pyserial ]));
```

Comment:
```suggestion
      python = lib.getExe (python.withPackages [ pyserial ]);
```

Reply:
Shouldn't we use the one that comes with the python environment through `ps:`? Or does it make a difference here?

Reply:
`python` designates exactly the one that matches your environment, I think, so these should be equivalent?

Reply:
It actually doesn't make a difference because any `packageOverrides` would be present in `python.pkgs` as well. So I guess either is fine.